CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to the Oasis Playground
+
+Developers are encouraged to add their dApp to the Oasis Playground. In order
+to list your dApp, follow the steps:
+
+1. Make sure your dApp is licensed under the one of the [OSI-approved]
+   licenses.
+2. Fork this repository.
+3. Copy `examples/.template.yaml` to a new file inside the `examples/`
+   subdirectory and adapt it accordingly to best describe your dApp.
+4. Add screenshots of your dApp into `examples/screenshots/` subdirectory. Name
+   them as `<your-dApp>-<number>.png` where `your-dApp` is the name of the YAML
+   file (without the extension) and `number` is the order of the screenshot
+   that will be considered—The first screenshot will be shown on the landing
+   page and the rest in the dApp details page.
```

Comment:
I would make per folder projects, this can get quite messy with even 10 project listed, imagine 100. So `projects/` folder, `dapp1/` folder - inside `description.yaml` and `screenshot/` folder, and `example/` folder which can be copied to create a template for dapp.

Reply:
Good idea!